sympy/series/limits.py

```diff
@@ -109,6 +109,22 @@ def __new__(cls, e, z, z0, dir="+"):
         obj._args = (e, z, z0, dir)
         return obj
 
+
+    @property
+    def free_symbols(self):
+        e = self.args[0]
+        # e and z are same then none is free_symbols
+        if e is self.args[1]:
```

Comment:
I knew this,  but since currently `Limit(expr + 1, expr, a).doit()` raise `Not Implemented Error` which is the reason i did not consider currently these cases. Though adding that functionality would not be tough though ( i guess). After this argument, do you think i should also consider these `Limit(expr + 1, expr, a)` cases also ?
Currently
```
>>> Limit(x**2*y + x, x**2, 1).doit()
NotImplementedError
```